docs/source/contributing/setup.rst

```diff
@@ -75,6 +75,8 @@ happen.
 
       npm install -g configurable-http-proxy
 
+   Question: Is this supposed to say ACCESS instead of EACCES?
```

Comment:
EACCES is correct. it is the name of the error code (see `errno.EACCES` in Python)